var/spack/repos/builtin/packages/stat/package.py

```diff
@@ -57,8 +52,7 @@ class Stat(AutotoolsPackage):
     depends_on("libtool", type="build")
     depends_on("dyninst", when="@4.2.1:")
     depends_on("dyninst@:11.9", when="~dysect @:4.2.0")
-    depends_on("dyninst@:9", when="@:4.0.1")
-    depends_on("dyninst@8.2.1+stat_dysect", when="+dysect")
+    depends_on("dyninst +stat_dysect", when="+dysect")
```

Comment:
I've not tested stat yet. I don't think this will build, and the patch definitely won't apply to recent versions of Dyninst.

Reply:
I have to test this too. Do you have any knowledge of the `stat` package? Is that variant used / useful?

Reply:
dyninst@:9 won't build because it assumes rpc/rpc.h is in system dirs, isn't configurable, and not a dependency (e.g., libtirpc). I'm fine getting rid of that. In which case, that variant and configuration can be dropped from stat. I've not tested the rest of the versions. I'm looking at hpctoolkit right now. Here are my testing results for all versions of the dependent package using all versions of dyninst@10.0.0:. Feel free to update this if/when you have time.

| package  | status | notes |
| ------------- | ------------- | ------------- |
| callpath | OK | |
| cbtf-krell | OK | needs #47339 |
| extrae | OK | needs #47359 |
| mitos | FAIL | I'm working on a patch |
| openspeedshop | FAIL | DEAD- requires python2 but was removed from spack by c5883fffd7 (22-Nov-2022) |
| openspeedshop-utils | FAIL | (same) |
| hpctoolkit | in progress |  |
| must |  |  |
| stat |  |  |
| tau |  |  |